docs/socialaccount/providers/microsoft.rst

```diff
@@ -25,3 +25,7 @@ for the login. To restrict it, change the ``tenant`` setting as shown below.
           ]
       }
   }
+
+.. note:: When you have configured you application to use single tennant authentication make sure to use above fragment to set ``tenant`` value to ``organizations`` in order to prevent the following error:
```

Comment:
```suggestion
.. note:: When you have configured your application to use single tenant authentication make sure to use the fragment above to set the ``"tenant"`` value to ``"organizations"`` in order to prevent the following error:
```